src/components/App.js

```diff
@@ -103,6 +103,14 @@ export class App extends Component {
             authorization={this.props.authorization}
             verifyEmail={this.props.verifyEmail}
           />
+          <Route
+            path="/ssoLogin/:loginIdp"
+            component={({ match }) => {
+              if (match.params.loginIdp)
```

Comment:
I'd add the `{ .. }` with this `if` statement since it appears to be multi line?

Reply:
fixed in #35 , going to close it